sub-packages/bionemo-geneformer/src/bionemo/geneformer/scripts/train_geneformer.py

```diff
@@ -89,10 +89,19 @@ def main(
     nsys_end_step: Optional[int] = None,
     nsys_ranks: List[int] = [0],
     config_class: Type[BioBertConfig] = GeneformerConfig,
+    num_layers: int = 6,
+    hidden_size: int = 256,
+    ffn_hidden_size: int = 512,
+    num_attention_heads: int = 4,
     log_every_n_steps: int = 50,
     gc_interval: int = 0,
     aligned_megatron_ddp: bool = False,
     recompilation_check: bool = False,
+    layernorm_eps: float = 1e-12,
+    adam_eps: float = 1e-8,
+    adam_w: float = 0.01,
+    adam_beta2: float = 0.999,
+    grad_reduce_in_fp32: bool = False,
```

Comment:
We might want to switch to `adap_eps=1e-5` and `grad_reduce_in_fp32=True` in light of the wandb results.